include/verilated.h

```diff
@@ -2493,6 +2494,65 @@ static inline void VL_ASSIGNSEL_WIIW(int obits, int lsb, WDataOutP owp, WDataInP
     _VL_INSERT_WW(obits, owp, rwp, lsb + obits - 1, lsb);
 }
 
+//======================================================================
+// Variable randomization (std::randomize, randomize method)
+
+inline IData VL_RANDOMIZE_VAR(CData& odata, int obits) VL_MT_SAFE {
+    odata = VL_RANDOM_Q(obits);
+    return 1;
+}
```

Comment:
Can we get rid of these functions?  Just emit the sets directly.  Much cleaner and speeds up compiling.

Reply:
I added these functions because I think this way it'll be easier to do constraints and `std::randomize` in the future. I guess if I remove this, I may as well get rid of AstStdRandomize and just generate C calls.

Reply:
Constraints will require looking at all of the variables together, so I don't see how this would help that, would think makes case for a more Verilator-directed approach more important, e.g. a given randomize might need to call different functions.